python/lsst/meas/extensions/gaap/gaap.py

```diff
@@ -0,0 +1,186 @@
+#
+# LSST Data Management System
+# Copyright 2008-2020 AURA/LSST.
```

Comment:
New files should not include copyright attribution in their license preambles.  See https://developer.lsst.io/stack/license-and-copyright.html#license-preambles-in-source-files, and please check all of the other new files.